src/cow-react/modules/limitOrders/containers/TradeButtons/limitOrdersTradeButtonsMap.tsx

```diff
@@ -89,7 +93,9 @@ export const limitOrdersTradeButtonsMap: { [key in LimitOrdersFormState]: Button
   [LimitOrdersFormState.QuoteError]: ({ quote }: TradeButtonsParams) => {
     return (
       <SwapButton disabled={true}>
-        <Trans>{quote.error ? quoteErrorTexts[quote.error.type] : 'Unknown error'}</Trans>
+        <Trans>
+          {quote.error ? quoteErrorTexts[quote.error.type] || 'Error loading price. Try again later.' : 'Unknown error'}
```

Comment:
I'm not sure i like `Unknown error` too much.  If this is an error getting the quote, wouldn't it better to say so? Same message as you shouw there sounds ok to me `'Error loading price. Try again later.'`